02week/pigLatin.js

```diff
@@ -7,13 +7,32 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
-
+const vowels = ['a', 'e', 'i', 'o', 'u']
 function pigLatin(word) {
-
-  // Your code here
-
+  word = word.trim().toLowerCase().split('');
+  for (let i = 0; i < word.length; i++) {
+    if (vowels.indexOf(word[i]) !== -1 && word[i] === word[0]) {
```

Comment:
The later part of this if statement doesn't make sense. Why does it matter if the first letter of the word is equal to the current index?